src/routers/secure/scrape.py

```diff
@@ -290,7 +292,7 @@ def get_info_hash(magnet: str) -> str:
         "message": "Started manual scraping session",
         "session_id": session.id,
         "torrent_id": torrent_id,
-        "torrent_info": torrent_info,
+        "torrent_info": torrent_info.dict(),
```

Comment:
I fixed the api issues in pr #915 

just waiting for it to get merged

Reply:
I see that's in draft, with multiple comments from reviewer. Is there a timeframe you are looking to merge ?

Reply:
Sometime today #915 should be merged. Then we can get this one in as well. Lets remove the api changes here.

Reply:
lets wait for raise, will rebase on updated mainline

Reply:
updated mainline, if you want to rebase and tweak 👌